internal/routes/all_routes.go

```diff
@@ -36,6 +48,8 @@ func apiRouter() http.Handler {
 		})
 	})
 
+	r.Get("/openapi.json", s.ServeOpenAPISpec)
```

Comment:
this is still being gated by the identity enforcement, but if you do not see a good way of not gating it, I guess we can keep it for now.
But I do not like the position of this route. I think it should be first route, but for sure it looks weird if its in the middle, looks kinda random.

Reply:
we have two routes for the spec file, one is gated by the identity (`<pathPrefx>/openapi.json`) for consoleDot usage, and the "upstream" use (`<root>/spec.json`) which is not gated by it.
The docs (redoc) endpoint is not gated by the identity either (`<root>/docs`)